VideoLocker/src/main/java/org/edx/mobile/player/PlayerController.java

```diff
@@ -763,7 +766,9 @@ public void onClick(View v) {
 //          }
         }
     };
-
+    /**
+     * Currently unused 3/4/2015

```

Comment:
I think we can complete this code after addressing the comment below about not using a boolean.